frontend/src/metabase/ui/components/overlays/Popover/index.tsx

```diff
@@ -6,15 +6,12 @@ export { getPopoverOverrides } from "./Popover.styled";
 
 const MantinePopoverDropdown = Popover.Dropdown;
 
-const PopoverDropdown = Object.assign(function PopoverDropdown(
-  props: PopoverDropdownProps,
-) {
+const PopoverDropdown = function PopoverDropdown(props: PopoverDropdownProps) {
```

Comment:
This is only a refactor. I removed unnecessary `Object.assign` and type casting.